rstudio/c9s-python-3.9/Dockerfile

```
@@ -50,6 +50,7 @@ COPY rsession.conf /etc/rstudio/rsession.conf
 # package installation 
 RUN dnf install -y libsodium-devel.x86_64 libgit2-devel.x86_64 libcurl-devel harfbuzz-devel.x86_64 fribidi-devel.x86_64 cmake "flexiblas-*"
 RUN R -e "install.packages('Rcpp')"
+RUN if [-n = $NEEDPACKAGE]; then R -e "install.packages(c('tidyverse', 'tidymodels', 'plumber', 'vetiver', 'devtools'))" && R -e "install_versions('tidyverse', version = '2.0.0', 'tidymodels', version = '1.1.1', 'vetiver', version = '0.2.5', 'plumber', version = '1.2.1', 'devtools', verison = '2.4.5' )"; else exit 0
```

Does this really work? :thinking: I would rather expect something like this:

```
if [ -n "$NEEDPACKAGE" ];
```

Also it would be nice to put some echo in case we skip this installation. And I'm not sure if we want exit there? :thinking: 

```suggestion
RUN if [ -n "$NEEDPACKAGE" ]; then R -e "install.packages(c('tidyverse', 'tidymodels', 'plumber', 'vetiver', 'devtools'))" && R -e "install_versions('tidyverse', version = '2.0.0', 'tidymodels', version = '1.1.1', 'vetiver', version = '0.2.5', 'plumber', version = '1.2.1', 'devtools', verison = '2.4.5' )"; else echo "The variable NEEDPACKAGE not set -> skipping installation of extra R packages"; fi
```

Disclaimer: I didn't really tested it in any way :slightly_smiling_face:

---

Thanks. Resolving this for now.